instrumentation/opentelemetry-instrumentation-sqlalchemy/src/opentelemetry/instrumentation/sqlalchemy/engine.py

```diff
@@ -27,6 +27,16 @@
 from opentelemetry.trace.status import Status, StatusCode
 
 
+def _sanitize_query(query):
+    """Remove query content, replace with sanitization symbol.
+    For example `SELECT * FROM table` will sanitize to SELECT ? FROM ?`
+    """
+    sanitize_symbol = " ?"
+    if query and query.split():
+        return query.split()[0] + sanitize_symbol + " FROM" + sanitize_symbol
+    return ""
```

Comment:
This logic is incorrect for many cases, right? Just taking the first word and concatenating it with `? FROM ?` because statements may contain multi-word instructions such as `INSERT INTO` and `ALTER TABLE`, and it's not always guaranteed that there will be `FROM`.